src/main/java/emu/grasscutter/game/quest/exec/ExecRefreshGroupSuite.java

```diff
@@ -22,17 +23,23 @@ public boolean execute(GameQuest quest, QuestData.QuestExecParam condition, Stri
 
             var scriptManager = quest.getOwner().getScene().getScriptManager();
 
-            quest.getMainQuest().getQuestGroupSuites().add(QuestGroupSuite.of()
+            // mainly trying to avoid unlimited incrementation group suites that get saved to DB
+            // but the following implementation sometimes throws error, and cause rewind to fail
+            // in game and require login and out to work
+            if (!quest.getMainQuest().getGroupSuitesTracker().contains(quest.getSubQuestId())) {
+                quest.getMainQuest().getGroupSuitesTracker().add(quest.getSubQuestId());
+                quest.getMainQuest().getQuestGroupSuites().add(QuestGroupSuite.of()
                 .scene(sceneId)
                 .group(groupId)
                 .suite(suiteId)
                 .build());
+            }
 
             // refresh immediately if player is in this scene
             if (quest.getOwner().getScene().getId() == sceneId) {
                 var targetGroup = scriptManager.getGroupById(groupId);
                 if (targetGroup == null) {
-                    Grasscutter.getLogger().warn("trying to load unknown group {} in scene {}", groupId, sceneId);
+                    Grasscutter.getLogger().debug("trying to load unknown group {} in scene {}", groupId, sceneId);
```

Comment:
This should probably stay a warning, since it will likely result in wrong behavior afterwards

Reply:
I changed it because it was annoying when i was debugging for the other stuff, forgot to change it back after that